.circleci/config.yml

```diff
@@ -18,31 +17,36 @@ commands:
       - run: echo $(python --version) > .python-version
       - restore_cache:
           key: py-deps-{{ checksum ".python-version" }}-{{ checksum "docs/requirements.txt" }}
-      - run: "if [ ! -d ./venv ]; then python -m venv ./venv; fi"
+      - run: 'if [ ! -d ./venv ]; then python -m venv ./venv; fi'
       - run: echo "source $(pwd)/venv/bin/activate" >> $BASH_ENV
       - run: pip install -r ./docs/requirements.txt
       - save_cache:
           key: py-deps-{{ checksum "docs/requirements.txt" }}
           paths:
             - ./venv
 
-
 aliases:
   - &node12
     image: circleci/node:12
 
-  - &python-docs
-    # using Python 3.6 as that's the version ReadTheDocs is running
+  - &python-docs # using Python 3.6 as that's the version ReadTheDocs is running
     image: circleci/python:3.6-node
 
   - &test-steps
     steps:
       - checkout
       - npm-install
+      - run: npm run build
       - run: npm run ci:test
 
-
 jobs:
+  build:
```

Comment:
Does it need to be tested separately if it's a part of all the other jobs?

Reply:
Besides, this job isn't included in the workflow, so it isn't currently being ran.